phpstan.neon.dist

```diff
@@ -0,0 +1,330 @@
+parameters:
+    level: 2
+    paths:
+        - ./
+    excludes_analyse:
+        - cache/
+        - build/
+        - vendor/
+
+    # TODO review once we drop PHP 7.x support
+    treatPhpDocTypesAsCertain: false
+
+    ignoreErrors:
+        - '~^Unsafe usage of new static\(\)\.$~'
+
+        # TODO these rules are generated, this ignores should be fixed in the code
```

Comment:
we will merge now and every newly introdues error is checked

any help (later, in another PR) is welcomed